src/model/a320-interior/A320_NEO_INTERIOR_LOD00.gltf

```diff
@@ -187213,7 +187213,6 @@
                 66,
                 67,
                 70,
-                73,
```

Comment:
Shouldn't you remove it on all LODs?

Reply:
I decided to just hide it using modelbehaviors so I don't have to modify any original model files.